ETC.tsv

```diff
@@ -10060,7 +10060,7 @@ ETC_20150323_010061	$Don't lie! I will finish you too!
 ETC_20150323_010062	$You think I'll forgive you? 
 ETC_20150323_010063	$I will destroy you too!
 ETC_20150323_010064	$You think I'll fall for that? Come on and fight!
-ETC_20150323_010065	$Picking flower...
+ETC_20150323_010065	$Picking Flower...
```

Comment:
I have this as: Picking a Flower

The original doesn't have the unnecessary extra periods.